scripts/release_promote_stable.sh

```diff
@@ -4,6 +4,9 @@ set -euo pipefail
 # shellcheck source=scripts/lib.sh
 source "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
 
+# Make sure GITHUB_TOKEN is set for the release command.
```

Comment:
This only works when `CODER=true` i.e. inside a Coder workspaces.

Reply:
Do we suggest adding an extra "echo" to indicate it?

Reply:
That’s the expected use-case. And if it isn’t set outside a workspace it’ll print a notice to do `gh auth login`. Just realized that won’t work though, but we could update `lib.sh` to fetch the token via `gh auth token` in this case.

Reply:
daf03f8d70cec593d3b39d4b0245940e5c32ff4e

Reply:
Perhaps we also shouldn't override GITHUB_TOKEN if it's set? And maybe support GH_TOKEN too (in lib.sh)? Thoughts @matifali?

Reply:
Yes as far it works and make it more robust. Let's do this. I don't fully understand the difference between `GH_` and `GITHUB_` prefix for tokens.

Reply:
Yes please go ahead. Not blocking you. It looks perfect.